### PR TITLE
Notify the user when a friend comes online and goes offline

### PR DESCRIPTION
**Pull request by endoffile78**

### CODE REVIEW

File: src/main.h [hunk 1]
```diff
@@ -122,6 +122,7 @@ typedef struct {
     uint8_t push_to_talk : 1;
     uint8_t use_mini_roster : 1;
     uint8_t group_notifications : 4;
+    uint8_t status_notifications: 1;
     uint8_t zero : 2;
```
Comment (GrayHatter) [inline src/main.h]: Don't forget to change this bit field too. I.e. DO NOT, change the size of this struct!

http://stackoverflow.com/a/3186014

**Comment by GrayHatter** [comment]
@uTox/translators

**Comment by sfan5** [comment]
de.h:
```c
msgid(STATUS_NOTIFICATIONS)
msgstr("Statusbenachrichtigungen")
```

**Comment by endoffile78** [comment]
@GrayHatter  I think that would be good. Ill add a notifications section.

**Comment by GrayHatter** [comment]
@endoffile78 ping me once you rebase and finish this. If you're able to finish the UI reorg I'll tag and release a new version of uTox

**Pull request merged**